<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradingOptionsDialog</class>
 <widget class="QDialog" name="GradingOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grading Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#GradingOptionsDialog{
background:qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.511364, stop:0 rgba(50, 117, 116,255) , stop:1 rgba(69, 135, 133,255) )
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save_options_button">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#save_options_button{
color:white;
background:rgb(25, 150, 141);
border-color:rgb(25, 150, 141);
border-radius: 5px;
height: 25px
}

QPushButton#save_options_button:hover
{
color:(220, 223, 223);
background:rgb(21, 156, 143);
border-color:rgb(21, 156, 143);
border-radius: 5px;
height: 25px
}

QPushButton#save_options_button:pressed
{
color:rgb(211, 213, 213);
background:rgb(16, 117, 107);
border-color:rgb(16, 117, 107);
border-radius: 5px;
height: 25px
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#ok_button{
color:white;
background:rgb(25, 150, 141);
border-color:rgb(25, 150, 141);
border-radius: 5px;
height: 25px
}

QPushButton#ok_button:hover
{
color:(220, 223, 223);
background:rgb(21, 156, 143);
border-color:rgb(21, 156, 143);
border-radius: 5px;
height: 25px
}

QPushButton#sok_button:pressed
{
color:rgb(211, 213, 213);
background:rgb(16, 117, 107);
border-color:rgb(16, 117, 107);
border-radius: 5px;
height: 25px
}</string>
       </property>
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="max_dist_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>All the nearest
 neighbors 
(distances ≤2%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="min_lab_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white</string>
     </property>
     <property name="text">
      <string>Minimum of 
 laboratories deposited 
sequences
(minimun ≥2)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="min_sequences_label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white
</string>
     </property>
     <property name="text">
      <string>Sequences deposited 
or
published independently
(minimum ≥ 3)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="max_dist_text">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="min_lab_text"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="min_sequences_text"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
